### ISSUE TITLE
1.4.10 - RDP - Add option for session width and height to match client area

### ISSUE THREAD

**Issue by waukeerules** [jimradford/superputty#845]
Works great otherwise; I managed to find a work around to get the RDP window to match my screen area of SuperPuTTY by using "Extra Arguments" to manually set the width and height. Would be nice to add that as an option in Tools > Options that way it's not managed per session. 

Extra Arguments: /w:1448 /h:892

![image](https://user-images.githubusercontent.com/12787084/142790586-f2984762-76e6-43f5-b0c3-072bc541d0f8.png)

**Comment by jimradford** [comment]
I agree, SuperPuTTY should detect the client area and send the correct parameters so it fills the window area automatically unless someone uses the extra arguments parameters to override it. RDP support is still a bit sketchy at this point.

**Comment by waukeerules** [comment]
The only other "want" would be a default username template, but that would be an interesting item to implement. I generally use "Remote Desktop Connection Manager v2.7" which has the capability. So do-able somehow, just a bit of effort to implement I am sure.
